watertap/flowsheets/crystallization/Crystallizer_MVR.py

```diff
@@ -0,0 +1,444 @@
+#################################################################################
```

Comment:
Next step is to add (1) a test file for this flowsheet and (2) documentation for this flowsheet in this section: https://watertap.readthedocs.io/en/latest/technical_reference/flowsheets/index.html